mjml/core/registry.py

```diff
@@ -1,10 +1,14 @@
 
 
-__all__ = []
+__all__ = ['assign_components', 'components', 'handle_mjml_config_components', 'preset_core_components', 'register_component']
+
+import inspect
+from types import ModuleType
+from typing import Type
 
 components = {}
 
-def _components():
+def preset_core_components():
```

Comment:
How about `register_core_components()`? To me that name would indicate that calling the function not only returns the preset components but puts them in the registry as well.